scanner/cmd/scanner/main.go

```diff
@@ -25,16 +25,17 @@ import (
 	"github.com/stackrox/rox/scanner/indexer"
 	"github.com/stackrox/rox/scanner/internal/version"
 	"github.com/stackrox/rox/scanner/matcher"
+	"github.com/stackrox/rox/scanner/services"
 	"golang.org/x/sys/unix"
 )
 
 // Backends holds the potential engines the scanner
 // may use depending on the mode in which it is running.
 type Backends struct {
 	// Indexer is the indexing engine.
```

Comment:
I think both me and @RTann were suggesting to use a different name for backends but I don't remember clearly. My suggestion was to use `scannerComponents`

Reply:
I think `Backend` is suitable:

![image](https://github.com/stackrox/stackrox/assets/291493/18837f67-99cc-483e-8fd4-067f6f049211)

In the case Scanner, the "user" is Central and Sensor.